### DISCUSSION TITLE
[@types/faker] Deprecate the package

### DISCUSSION THREAD

**Discussion by damienwebdev** [DefinitelyTyped/DefinitelyTyped#58206] [Issues with a @types package]
Hi! I'm one of the new maintainers of https://github.com/faker-js/faker (the new home of the former https://github.com/marak/faker.js).

We re-released the old `faker` versions exactly (sans the package name change) as was published before Marak deleted his project.

Prior releases required the use of `@types/faker`, but new ones [(as of 6.0.0-alpha.3)](https://github.com/faker-js/faker/releases/tag/v6.0.0-alpha.3 ) do not.

> Typescript Support
> Types are available via the @types/faker package. You must manually link them using a *.d.ts file, like so:

> ```
> // faker.d.ts
> declare module '@faker-js/faker' {
>   import faker from 'faker';
>   export default faker;
> }
> ```

We just shipped https://github.com/faker-js/faker/releases/tag/v6.0.0-alpha.3 which includes support for first-party emitted types, see the example here: https://stackblitz.com/edit/typescript-wsqtmx?file=index.ts

We would like to "deprecate" `@types/faker` and encourage users to update to the `v6` line. I don't know the mechanics of what this means to DefinitelyTyped, so any guidance would be appreciated!

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
Hi, @damienwebdev. I created PR to remove the removed package from DT #58218
